src/components/film-card/film-card.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Link} from 'react-router-dom';
+
+const FilmCard = (props) => {
+  const {
+    film: {
+      previewImg,
+      name
+    },
+    handleFilmCardClick,
+    handleFilmCardMouseOver
+  } = props;
+
+  return (
+    <article
+      className="small-movie-card catalog__movies-card"
+      onClick={handleFilmCardClick}
+      onMouseOver={handleFilmCardMouseOver}
+    >
+      <div className="small-movie-card__image">
+        <img src={previewImg} alt={name} width="280" height="175" />
+      </div>
+      <h3 className="small-movie-card__title">
+        <Link to="/films/:id" className="small-movie-card__link">
+          {name}
+        </Link>
+      </h3>
+    </article>
+  );
+};
+
+FilmCard.propTypes = {
+  handleFilmCardClick: PropTypes.func.isRequired,
+  handleFilmCardMouseOver: PropTypes.func.isRequired,
+  film: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    previewImg: PropTypes. string.isRequired
```

Comment:
Вообще непонятно как с такими ошибками проходит проверка линтера... 🤷 

```suggestion
    previewImg: PropTypes.string.isRequired
```

Reply:
🤷
Поправил